### ISSUE TITLE
numpy-1.8.0.tar.gz is not archive

### ISSUE THREAD

**Issue by levrusha** [amusecode/amuse#194]
i try `./install.py install`
i get

```
Fortran 90 compiler used will be:  gfortran
Fortran 77 compiler used will be:  gfortran
Files are installed in:  /home/valeria/amuse/prerequisites
Files are downloaded to:  /home/valeria/amuse/prerequisites/install/_temp
Unpacking  numpy-1.8.0.tar.gz
starting  tar -xf numpy-1.8.0.tar.gz
tar: Это не похоже на tar-архив
gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
----------------------------------------------------------
Could not unpack source file of numpy
----------------------------------------------------------
Download location may have changed
Please download the source file yourself, 
or contact the AMUSE development team.
http://castle.strw.leidenuniv/trac/amuse
To download the file you can update the URL in
one of the following lines and run the command.
curl  http://pypi.python.org/packages/source/n/numpy/numpy-1.8.0.tar.gz -o /home/valeria/amuse/prerequisites/install/_temp/numpy-1.8.0.tar.gz
or
wget  http://pypi.python.org/packages/source/n/numpy/numpy-1.8.0.tar.gz -O /home/valeria/amuse/prerequisites/install/_temp/numpy-1.8.0.tar.gz
Note: The name of the output file must not be changed (after the -o or -O parameter)
```

i try
`wget  http://pypi.python.org/packages/source/n/numpy/numpy-1.8.0.tar.gz -O /home/valeria/amuse/prerequisites/install/_temp/numpy-1.8.0.tar.gz`

get
```
wget: /home/valeria/amuse/prerequisites/lib/libcrypto.so.1.0.0: no version information available (required by wget)
wget: /home/valeria/amuse/prerequisites/lib/libssl.so.1.0.0: no version information available (required by wget)
wget: /home/valeria/amuse/prerequisites/lib/libssl.so.1.0.0: no version information available (required by wget)
--2018-03-11 13:35:51--  http://pypi.python.org/packages/source/n/numpy/numpy-1.8.0.tar.gz
Распознаётся pypi.python.org (pypi.python.org)... 151.101.84.223, 2a04:4e42:14::223
Подключение к pypi.python.org (pypi.python.org)|151.101.84.223|:80... соединение установлено.
HTTP-запрос отправлен. Ожидание ответа... 403 SSL is required
2018-03-11 13:35:52 ОШИБКА 403: SSL is required.
```

if i run
`curl  http://pypi.python.org/packages/source/n/numpy/numpy-1.8.0.tar.gz -o /home/valeria/amuse/prerequisites/install/_temp/numpy-1.8.0.tar.gz`

it did not help

**Comment by rieder** [comment]
The numpy archive seems fine to me. 
It looks like maybe something else is going on on your system. The error you get when running wget points in that direction as well. 
What is the output you get when running the curl command?

**Comment by levrusha** [comment]
when I use curl, I see the time for which it downloaded
when i see this file in folder _temp, this file numpy is not archive, it is text file

**Comment by rieder** [comment]
Thanks. The URL is wrong, it should be https instead of http. 
This URL is correct in the current AMUSE version, I think you're using an older version.
Can you try again with the most recently available version (v11.2, or the current git version)?
